model/src/main/proto/oppia_logger.proto

```diff
@@ -120,7 +120,33 @@ message EventLog {
   }
 }
 
-// Structure of a container that contains event logs.
-message EventLogs{
+// Structure for an exception log.
+message ExceptionLog {
```

Comment:
Do we also need to log the exception's stack trace?

Reply:
The exception offline logging functionality is going to be part of M3. Stack trace is something that Sarthak needs to figure how to serialize. So I said that he can figure that out while actually implementing M3 and for now he doesn't need to add it.
If we do separate out the logic for both then we can completely remove these proto files and keep only the event ones for now

Reply:
Removed ExceptionLog related proto files.